### DISCUSSION TITLE
Welcome to Vulnerator Discussions!

### DISCUSSION THREAD

**Discussion by amkuchta** [Vulnerator/Vulnerator#210] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Lightning454** [comment]
Love your tool.  I was wondering if you were going to incorporate NIST 800-53 Rev 5 in a future release of Vulnerator?

**Reply by ebarlowjr2** [reply]
Do you currently use this tool? I have not been able to get in touch with the creator and im currently in the process of looking for developers to take over the project.  What industry are you in if that not too much info?
